frontend/src/components/Footer/footer.tsx

```diff
@@ -0,0 +1,36 @@
+import {Typography, useTheme, Grid} from '@mui/material';
+
+export default function Footer(props: {Logo: string | undefined}) {
+  const theme = useTheme();
+
+  return (
+    <Grid
+      container
+      style={{
+        justifyContent: 'center',
+        alignItems: 'center',
+        margin: '0',
+        padding: '0',
+      }}
+    >
+      <Grid item p={0.8}>
+        <img
+          src={props.Logo}
+          alt="College-logo"
+          style={{height: 'calc(8px + 1vw)'}}
+        ></img>
+      </Grid>
+      <Grid item>
+        <Typography
```

Comment:
Text is quite small for both desktop and mobile screens. Also, it is not center aligned with the logo.
Mobile screen example:
![image](https://github.com/bsoc-bitbyte/busify/assets/68912239/d29a9c44-372e-41e3-a582-b7ee6a0201a7)

Reply:
@prajjwalkapoor   I just wanted to confirm that do you want the footer section to look as the image  attached above when viewing  it in mobile screen?